### ISSUE TITLE
default destination folder

### ISSUE THREAD

**Issue by vipertongn** [mmozeiko/RcloneBrowser#122]
is there a way to set up a default destination folder for downloads without having to input the destination after every download? Also a method to batch select certain files to download from a drive?

**Comment by imthenachoman** [comment]
why not use the actual rclone command to do that?

**Comment by mastahcez** [comment]
@imthenachoman You mean there's a way to set a default local destination setting somewhere in rclone?

**Comment by imthenachoman** [comment]
No, I am saying if you want to do batch downloads, why not script something with `rclone`?
